guides/micronaut-oracle-autonomous-db/micronaut-oracle-autonomous-db.adoc

```diff
@@ -143,25 +143,29 @@ resource:application-oraclecloud.yml[]
 
 Create a test to verify that database access works:
 
-test:ThingRepositoryTest[]
+test:repository/ThingRepositoryTest[]
 
 == Testing the Application
 
 There are two options for running the tests; one is to run against the live database, and the other is to run tests locally with an Oracle database in a Docker container using https://www.testcontainers.org/[Testcontainers].
 
-Add the following dependencies:
+Replace the following dependencies:
```

Comment:
minor - change "dependencies" to "dependency", and remove both `:dependencies:` - that's only needed if you have 2 or more dependencies that you want to render together